src/sentry/snuba/metrics/datasource.py

```diff
@@ -320,7 +330,15 @@ def _fetch_tags_or_values_per_ids(
     # entity by validating that the ids of the constituent metrics all lie in the same entity
     supported_metric_ids_in_entities = {}
 
-    for metric_type in ("counter", "set", "distribution"):
+    release_health_metric_types = ("counter", "set", "distribution")
+    performance_metric_types = ("generic_set", "generic_distribution")
```

Comment:
Is there any reason for not having `generic_counter`?

Reply:
Just because it was not supported by `METRIC_TYPE_TO_ENTITY` map. Added it now!